oracle_fdw.c

```diff
@@ -2855,6 +2856,8 @@ char
 			else if (fdwState->oraTable->cols[i]->oratype == ORA_TYPE_TIMESTAMPLTZ)
 				/* convert TIMESTAMP WITH LOCAL TIME ZONE to TIMESTAMP WITH TIME ZONE */
 				format = "%s(%s%s AT TIME ZONE sessiontimezone)";
+			else if (fdwState->oraTable->cols[i]->oratype == ORA_TYPE_NCLOB)
+				format = "%s(TO_CLOB(%s%s))";
```

Comment:
Seems like the parenthesis around `TO_CLOB()` aren't really needed, I will get rid of them.